its/core-it-support/core-it-plugins/maven-it-plugin-core-stubs/maven-ear-plugin/src/main/java/org/apache/maven/plugin/coreit/EarMojo.java

```diff
@@ -69,16 +68,6 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         if (!outputFile.isAbsolute()) {
             outputFile = new File(project.getBasedir(), pathname).getAbsoluteFile();
         }
-
-        getLog().info("[MAVEN-CORE-IT-LOG] Creating output file: " + outputFile);
-
-        try {
-            outputFile.getParentFile().mkdirs();
-            outputFile.createNewFile();
-        } catch (IOException e) {
-            throw new MojoExecutionException("Output file could not be created: " + pathname, e);
-        }
-
-        getLog().info("[MAVEN-CORE-IT-LOG] Created output file: " + outputFile);
```

Comment:
having no error implies its done. 
same philosophy in:
no complain is prais enough.